public/app/plugins/datasource/loki/backendResultTransformer.test.ts

```diff
@@ -39,12 +39,11 @@ const inputFrame: DataFrame = {
     {
       name: 'labels',
       type: FieldType.string,
-      config: {
-        custom: {
-          json: true,
-        },
-      },
-      values: new ArrayVector(['{ "level": "info", "code": "41🌙" }', '{ "level": "error", "code": "41🌙" }']),
+      config: {},
+      values: new ArrayVector([
+        { level: 'info', code: '41🌙' },
+        { level: 'error', code: '41🌙' },
+      ]),
```

Comment:
this unrelated change just fixes an unrelated test-case (data-format changed in the past but i forgot to update the test case)